docs/source/doc-conda-requirements.yml

```diff
@@ -0,0 +1,17 @@
+name: httomo
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.9
+  - sphinx
+  - sphinx_rtd_theme
+  - nbsphinx
+  - pandoc
+  - jinja2
+  - sphinx-panels
+  - sphinx-copybutton
+  - tomopy
```

Comment:
why do we need tomopy here?

Reply:
I wonder... If we get the templates locally then we don't really need it?

Reply:
Yes, I think it doesn't need to be there anymore. I think I added it maybe when I was running the python file templates/tomopy/yaml_generator_tomopy.py inside the github action, then I forgot to remove it.

Reply:
yes, thanks Jessica, would be nice if you remove it - the size is huge.